pre/remap_restart/remap_base.py

```diff
@@ -33,3 +33,44 @@ def remove_merra2(self):
     if self.config['input']['shared']['MERRA-2']:
       print(" remove temporary folder that contains MERRA-2 archived files ... \n")
       subprocess.call(['/bin/rm', '-rf', self.config['input']['shared']['rst_dir']])
+
+  def copy_without_remap(self, restarts_in, compared_file_in, compared_file_out, suffix):
+     config = self.config
+     in_agrid        = config['input']['shared']['agrid']
+     out_agrid       = config['output']['shared']['agrid']
+     out_levels      = config['output']['air']['nlevel']
+     in_bc_version   = config['input']['shared']['bc_version']
+     out_bc_version  = config['output']['shared']['bc_version']
+     in_stretch      = config['input']['shared']['stretch']
+     out_stretch     = config['output']['shared']['stretch']
+
+     if (in_agrid == out_agrid and in_stretch  == out_stretch):
+
+       for rst in restarts_in :
+         if 'fvcore_internal' in rst:
+           fvrst     = nc.Dataset(rst)
+           in_levels = fvrst.dimensions['lev'].size
+           if in_levels != int(out_levels): return False
+
+       cmd = 'diff -q ' + compared_file_in + ' ' + compared_file_out
+       print('\n' + cmd)
+       diff_str  = str(subprocess.check_output(shlex.split(cmd)))
+       if ' differ' in diff_str: return False
```

Comment:
@weiyuan-jiang : Thanks for the updated code.  I'm not sure I fully understand the logic.  When this function is called from `remap_catchANDcn.py`, presumably it only checks if the tile files differ.  If they are the same, then the catch restart would not be remapped.  But what if the tile files are the same and only the bcs version differs?  For example, a new set of bcs may differ from an older version only in the science values of, say, the snow albedo, with otherwise identical "geometry" (tile) files.  Maybe I'm missing something.